package.json

```diff
@@ -21,8 +21,8 @@
     "Srikanth Lakshmanan <srik.lak@gmail.com>"
   ],
   "devDependencies": {
-    "eslint-config-wikimedia": "^0.9.0",
-    "eslint-plugin-qunit": "3.3.1",
+    "eslint-config-wikimedia": "0.14.1",
```

Comment:
If we retain the caret ^ in the version number, installations can use minor version upgrades. Just checking if the removal of caret is intentional here and other upgrades.